setup.cfg

```diff
@@ -26,7 +26,9 @@ ignore =
 exclude = 
   src/porepy/__init__.py,
   src/porepy/numerics/ad/__init__.py,
-
+per-file-ignores =
```

Comment:
This is a relevant modification. I was introduced because of the imports on several __init__.py files.

Reply:
I think this is fine.